lib/rubygems/commands/sources_command.rb

```diff
@@ -34,6 +34,10 @@ def initialize
       options[:update] = value
     end
 
+    add_option '-f', '--[no-]force', 'Force prompted confirmations' do |value, options|
```

Comment:
Maybe a better help text would be "Automatically answer 'yes' to all confirmation prompts" or something like that?

Reply:
If you like, I will update.

I would suggest we stick with --force as an option though for consistency with other commands.

Reply:
Updated as suggested :+1:

Reply:
Yup, I only meant to update the help text, the name seems fine :+1:.